Classes/Response/YLPBaseObject.h

```diff
@@ -0,0 +1,15 @@
+//
+//  YLPBaseObject.h
+//  YelpAPI
+//
+//  Created by Brian Sharrief Alim Bowman on 11/7/16.
+//
+//
+
+@import Foundation;
+
+@interface YLPBaseObject : NSObject
+
++ (instancetype)new NS_UNAVAILABLE;
```

Comment:
yea its a pet peeve of mine to write the same code all over the place. I can adjust it however you like. The base may come in handy later if other functionality comes up that will be shared. I'll review the API changes and update sometime between tonight and tomorrow. Just let me know about other changes you like, i'll be happy to make them!